pkgconfig/CMakeLists.txt

```diff
@@ -78,18 +78,26 @@ set(ENCLAVE_CLIBS_1
     -Wl,-z,now
     -Wl,-gc-sections
     -L\${libdir}/openenclave/enclave
-    -loeenclave
-    -loecryptombedtls
-    -lmbedx509
-    -lmbedcrypto)
+    -loeenclave)
+
+set(ENCLAVE_MBEDTLS_CRYPTO_WRAPPER_LIBS -loecryptombedtls -lmbedx509 -lmbedtls
```

Comment:
We'd need to add any other library that mbedtls/openssl depend on to these lists. E.g oelibc, oecore. Otherwise if oelibc comes before oecryptombed in the linker line, there could be link errors.
@mingweishih Can you confirm?

Reply:
I think line number 95 should take care of that.